spec/steps/provision.fmf

```diff
@@ -68,3 +68,30 @@ example: |
             how: beaker
             family: Fedora31
             tag: released
+
+/connect:
+    summary: Connect to a provisioned box
+    description: |
+        Do not provision a new system. Instead, use provided
+        authentication data to connect to a running machine.
+
+        guest
+            hostname or ip address of the box
+        user
+            user name to be used to log in, ``root`` by default
+        password
+            user password to be used to log in
+        key
```

Comment:
Isn't this supposed to be `private-key-path`?

Reply:
afaik we decided to call it key, but the value is documented as a path, right @psss ?

Reply:
Yes, `key` is the option name and detailed description contains `private-key-path`.